.github/workflows/common-test.yaml

```diff
@@ -29,10 +29,11 @@ jobs:
           ${{ runner.os }}-build-codegen-
     - name: Checkout repository
       uses: actions/checkout@v2
-    - name: Setup golangci-lint
-      run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.25.0
     - name: Run golangci-lint
-      run: make lint
+      uses: golangci/golangci-lint-action@v2.5.2
```

Comment:
We should avoid the hardcode version checkin.

Reply:
dependabot should be scanning for action versions as well as library versions; it will open PRs to update the versions in CI workflows automatically (it couldn't do that previously because AFAIK it needs them in github `uses:` format rather than the shell script installation we had earlier).

not sure if that's on for all repos, but it probably is here, since the KIC repo usually has the most robust tooling setup

Reply:
Gotcha, "uses" are needed to enable dependabot auto scanning.